test/cypress/integration/SearchUserPage.spec.js

```diff
@@ -0,0 +1,20 @@
+describe('Search User Page', () => {
+	beforeEach(() => {
+		cy.visit('/search-users')
+		cy.injectAxe()
+	})
+
+	it('should load page', () => {
```

Comment:
Should there be additional assertions here? perhaps on the search input and button?

Reply:
I did attempt that but I ran in to fetch issues, something related to polyfill. So I created #49 to work on that later

Reply:
Gotcha - this is where you mentioned the other story to avoid scope creep. No problem, 😀